scss/forms/_form-check.scss

```diff
@@ -183,11 +188,11 @@
       }
 
       @if $enable-gradients {
-        background-image: $form-switch-checked-bg-image, var(--#{$prefix}gradient), var(--#{$boosted-prefix}switch-gradient);
+        --#{$prefix}form-switch-bg: #{escape-svg($form-switch-checked-bg-image)}, var(--#{$prefix}gradient), var(--#{$boosted-prefix}switch-gradient); // Boosted mod: extra param `var(--#{$boosted-prefix}switch-gradient)`
```

Comment:
Since we use CSS var now, we don't need the extra `var(--#{$boosted-prefix}switch-gradient)` anymore, so let it as the Bootstrap one. (same below)

Reply:
Fixed it in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/2f29505398b856e449a28273240603317d45b08c. I let you double-check that this modification is OK and if we don't need to remove this extra-param somewhere else.